src/databricks/labs/ucx/account.py

```diff
@@ -88,6 +88,42 @@ def sync_workspace_info(self):
                 path = f"{installation.path}/{self.SYNC_FILE_NAME}"
                 ws.workspace.upload(path, info, overwrite=True, format=ImportFormat.AUTO)
 
+    def create_account_level_groups(self):
+        """
+        Crawl all workspaces, and create account level groups if a WS local group is not present in the account
+        """
+        acc_groups = {}
+        ac_grp_ids = self._ac.groups.list(attributes="id")
```

Comment:
don't use abbreviations, at least for groups - it's two more characters and makes it more consistent